spec/integration/data_stream/manifest.spec.yml

```diff
@@ -32,6 +32,28 @@ spec:
             - 1
           - - array
             - 2
+    required_vars:
```

Comment:
This new field has not been added under "properties", so it is not used right now in data stream manifests.

As for instance, it is done with `vars`
https://github.com/elastic/package-spec/blob/8f1292454250c378af0432606bb99418cc19c753/spec/integration/data_stream/manifest.spec.yml#L524